src/contracts/core/DelegationManager.sol

```diff
@@ -535,137 +572,151 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
-     * @dev If the `operator` is indeed an operator, then the operator's delegated shares in the `strategies` are also decreased appropriately.
+     * @notice Removes `nonNormalizedShares` in `strategies` from `staker` who is currently delegated to `operator` and queues a withdrawal to the `withdrawer`.
+     * @dev If the `operator` is indeed an operator, then the operator's delegated nonNormalizedShares in the `strategies` are also decreased appropriately.
      * @dev If `withdrawer` is not the same address as `staker`, then thirdPartyTransfersForbidden[strategy] must be set to false in the StrategyManager.
      */
     function _removeSharesAndQueueWithdrawal(
         address staker, 
         address operator,
         address withdrawer,
         IStrategy[] memory strategies, 
-        uint256[] memory shares
+        uint256[] memory nonNormalizedShares
     ) internal returns (bytes32) {
         require(staker != address(0), "DelegationManager._removeSharesAndQueueWithdrawal: staker cannot be zero address");
         require(strategies.length != 0, "DelegationManager._removeSharesAndQueueWithdrawal: strategies cannot be empty");
     
-        // Remove shares from staker and operator
-        // Each of these operations fail if we attempt to remove more shares than exist
+        // Remove nonNormalizedShares from staker and operator
+        // Each of these operations fail if we attempt to remove more nonNormalizedShares than exist
         for (uint256 i = 0; i < strategies.length;) {
             // Similar to `isDelegated` logic
             if (operator != address(0)) {
                 _decreaseOperatorShares({
                     operator: operator,
                     staker: staker,
                     strategy: strategies[i],
-                    shares: shares[i]
+                    nonNormalizedShares: nonNormalizedShares[i]
                 });
             }
 
             // Remove active shares from EigenPodManager/StrategyManager
             if (strategies[i] == beaconChainETHStrategy) {
                 /**
-                 * This call will revert if it would reduce the Staker's virtual beacon chain ETH shares below zero.
+                 * This call will revert if it would reduce the Staker's virtual beacon chain ETH nonNormalizedShares below zero.
                  * This behavior prevents a Staker from queuing a withdrawal which improperly removes excessive
-                 * shares from the operator to whom the staker is delegated.
+                 * nonNormalizedShares from the operator to whom the staker is delegated.
                  * It will also revert if the share amount being withdrawn is not a whole Gwei amount.
                  */
-                eigenPodManager.removeShares(staker, shares[i]);
+                eigenPodManager.removeShares(staker, nonNormalizedShares[i]);
```

Comment:
Continuing my prev comment, this naming thing is also gonna be inconsistent. See here -- why is this not `epm.removeNonNormalizedShares`?

How do we know `removeShares` wants a non-normalized parameter?

yuck. let's please keep the existing `shares` concept as-is.